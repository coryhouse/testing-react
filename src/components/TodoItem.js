import React from 'react';

const TodoItem = props => {
  return (
    <div className="todo-item">
      <h2>{props.todo}</h2>
    </div>
  );
};

export default TodoItem;
