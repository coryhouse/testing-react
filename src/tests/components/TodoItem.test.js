import React from 'react';
import { shallow } from 'enzyme';
import TodoItem from '../../components/TodoItem';

test('should test TodoItem component', () => {
  const wrapper = shallow(<TodoItem todo="Read Newspaper" />);
  expect(wrapper).toMatchSnapshot();
});
