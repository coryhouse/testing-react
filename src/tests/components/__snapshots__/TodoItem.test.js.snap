// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test TodoItem component 1`] = `
<div
  className="todo-item"
>
  <h2>
    Read Newspaper
  </h2>
</div>
`;
